crates/eko/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[build-system]
+requires = ["maturin>=1.1,<2.0"]
+build-backend = "maturin"
+
+[project]
+name = "ekors"
+requires-python = ">=3.8"
+classifiers = [
+  "Programming Language :: Rust",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+]
+dependencies = ["cffi"]
+
+[tool.maturin]
+bindings = "cffi"
```

Comment:
Did we decide for CFFI eventually?

Reply:
Ok, let's say fine for the time being. I wanted to use PyO3, because we'd avoid a further dependency, and I believe it was possible to make it work with Numba. But it is a time-consuming task, it's not the best timing.